openshift_cli_installer/cli.py

```diff
@@ -211,34 +212,40 @@ def main(**kwargs):
     Create/Destroy Openshift cluster/s
     """
     kwargs.pop("pdb", None)
-    UserInput(**kwargs)
+    user_input = UserInput(**kwargs)
```

Comment:
Why user_imput saved?

Reply:
This user_input is passed to functions for set and unset GCP SA file, it requires parameters like `cluster.platform` and `gcp_service_account_file`

Reply:
you should use **kwargs, you should have all the info there

Reply:
updated the code to use `**kwargs` instead